api/swagger-spec/api.json

```diff
@@ -0,0 +1,27 @@
+{
+  "swaggerVersion": "1.2",
+  "apiVersion": "",
+  "basePath": "https://10.192.210.63:8443",
```

Comment:
I think we should probably support 127.0.0.1 sanely as a pure dev-test thing (for this scenario).  Either way, something we can fix post 3.0

----- Original Message -----
> > @@ -0,0 +1,27 @@
> > +{
> > +  "swaggerVersion": "1.2",
> > +  "apiVersion": "",
> > +  "basePath": "https://10.192.210.63:8443",
> 
> > Basically we don't really support --master=127.0.0.1 which means we can't
> > support --listen=127.0.0.1
> 
> I didn't realize this was a problem.  @liggitt did you know about this?  We
> don't have any validation preventing this.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2660/files#r31466440